pkg/transferer/transferer.go

```diff
@@ -164,7 +177,7 @@ func (t Transferer) writeChunksToFile(ctx context.Context, chunkC <-chan *dump.C
 	}
 }
 
-func (t Transferer) Export(ctx context.Context, pool ChunkPool) error {
+func (t Transferer) Export(ctx context.Context, lc *LoadChecker, pool ChunkPool) error {
```

Comment:
use an interface here to be able to test export process with a custom load checker (or disable it completely with a mock)

Reply:
resloved in 44c38c52f5f762ad3262986f5032da015d65fb52